app/src/main/res/values/colors.xml

```diff
@@ -110,6 +110,7 @@
   <color name="green_shade_40">#6644A85A</color>
   <color name="green_shade_10">#1A44A85A</color>
   <color name="light_green">#D9F3E7</color>
+  <color name="sub_text">#767775</color>
```

Comment:
@mschanteltc Could you help us here in getting the colour of the description of the Download permission. As per the Accessibility point of `The Contrast ratio of the text (foreground and background) inside the TextViews is 2.68 but the required one 4.50`

Download permission in the mocks is not clickable to get the colour of it. 
https://xd.adobe.com/view/e8aa4198-3940-47f9-514a-f41cc54457f6-9e9b/screen/12a7e001-fdf4-4e86-bf33-4d770438d2e3/specs/

Reply:
Let's change the subtext to #555555. LMK what contrast ratio that would make it and if it is sufficient enough.

Reply:
#555555 makes the ratio [7.45](https://contrast-ratio.com/#%23555555-on-%23ffffff), which seems fine. Also FYI #767775 makes the contrast ratio [4.5](https://contrast-ratio.com/#%23767775-on-%23ffffff).

Reply:
Ah thanks for the update. I would prefer that the RGB values stay the same, and since #666666 is used elsewhere in the app and is lighter than the one I mentioned, let's use that.